ui/src/js/storyTelling-page-storyEdit.js

```diff
@@ -14,17 +14,8 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     fluid.defaults("sjrk.storyTelling.page.storyEdit", {
         gradeNames: ["sjrk.storyTelling.page"],
         pageSetup: {
-            editorCssRules: {
-                hide: {
-                    "padding-top": "0",
-                    "grid-template": "none"
-                },
-                show: {
-                    "padding-top": "1rem",
-                    "grid-template-columns": "2fr 3fr"
-                }
-            },
-            hideProperty: "hidden"
+            hiddenEditorClass: "hidden",
+            hiddenEditorProperty: "hidden"
```

Comment:
It's odd to optionalize a html element attribute. What other options can this property be? I think you can remove this line and directly use "hidden" attribute at line 212.